Standards/scs-XXXX-vN-taxonomy-of-failsafe-levels.md

```diff
@@ -0,0 +1,182 @@
+| Kubelet software fault                       | Medium      | Crashing kubelet cannot start new pods on the node / kubelet might delete the pods or not / node marked unhealthy / replication controllers start new pods elsewhere |
+| Cluster operator error                       | Medium      | Loss of pods, services, etc. / lost of apiserver backing store / users unable to read API                                                                            |
+| Failure of multiple nodes or underlying DB   | Low         | Possible loss of all data depending on the amount of nodes lost compared to the cluster size, otherwise costly rebuild                                               |
+
+These failure scenarios can result in temporary (T) or permanent (P) loss of the resource or data within.
+Additionally, there are a lot of resources in IaaS alone that are more or less affected by these failure scenarios.
+The following tables shows the impact **when no redundancy or failure safety measure is in place**:
+
+### Impact on OpenStack Resources (IaaS layer)
+
+| Resource | Disk Loss | Node Loss | Rack Loss | Power Loss | Natural Catastrophy | Cyber Threat | Software Bug |
+|----|----|----|----|----|----|----|----|
+| Image | P (if on disk) | T (if on node) | T/P | T | P (T if lucky) | T/P | P |
+| Volume | P (if on disk) | T (if on node) | T/P | T | P (T if lucky) | T/P | P |
+| User Data on RAM /CPU | | P | P | P | P | T/P | P |
+| volume-based VM | P (if on disk) | T (if on node) | T/P | T | P (T if lucky) | T/P | P |
+| ephemeral-based VM | P (if on disk) | P | P | T | P (T if lucky) | T/P | P |
+| Ironic-based VM | P (all data on disk) | P | P | T | P (T if lucky) | T/P | P |
+| Secret | P (if on disk) | T (if on node) | T/P | T | P (T if lucky) | T/P | P |
+| network configuration (DB objects) | P (if on disk) | T (if on node) | T/P | T | P (T if lucky) | T/P | P |
+| network connectivity (materialization) | | T (if on node) | T/P | T | P (T if lucky) | T/P | T |
+| floating IP | P (if on disk) | T (if on node) | T/P | T | P (T if lucky) | T/P | T |
+
+### Impact on Kubernetes Resources (KaaS layer)
+
+:::note
+
+In case the KaaS layer runs on top of IaaS layer, the impacts described in the above table apply for the KaaS layer as well.
+
+:::
+
+| Resource | Disk Loss | Node Loss | Rack Loss | Power Loss | Natural Catastrophy | Cyber Threat | Software Bug |
+|----|----|----|----|----|----|----|----|
+|Node|P| | | | | |T/P|
+|Kubelet|T| | | | | |T/P|
+|Pod|T| | | | | |T/P|
+|PVC|P| | | | | |P|
+|API Server|T| | | | | |T/P|
+
+For some cases, this only results in temporary unavailability and cloud infrastructures usually have certain mechanisms in place to avoid data loss, like redundancy in storage backends and databases.
```

Comment:
These sentences are IaaS-specific I think. They belong under the table "Impact on OpenStack Resources (IaaS)"